extensions/blocks/podcast-player/edit.js

```diff
@@ -147,7 +156,7 @@ const PodcastPlayerEdit = ( {
 		if ( ! isSelected && isInteractive ) {
 			setIsInteractive( false );
 		}
-	}, [ isSelected ] );
+	}, [ isInteractive, isSelected ] );
```

Comment:
Added by the linter, it looks like this was missing.

Reply:
Yep. Any dependencies need to be listed. It will be the exhaustive deps rule I expect.

Reply:
This was intentional. In this case, we do not want this to be run when `isInteractive` changes. The purpose of this function is to only run when selection changes. 

Since this has been added the effect will be always run twice now — the first time when `isSelected` changes, which updates the `isInteractive` and that causes the second run :(

Reply:
I guess we should make sure all of us are linting/prettifying all the time.

Reply:
@marekhrabe If this is intentional, should we add a linter exemption here? Otherwise, this will keep sneaking back in.

Reply:
I've seen no linter complaint about this so I have no idea. What does the error look like?

Reply:
Added a linter exemption for now